src/compiler/watchPublic.ts

```diff
@@ -706,6 +712,10 @@ namespace ts {
             }
         }
 
+        function watchExtendedConfigFile(extendedConfigFile: string) {
+            return watchFile(extendedConfigFile, scheduleProgramReload, PollingInterval.High, watchOptions, WatchType.ConfigFile);

```

Comment:
You want different watch type so we can identify in log that its extended config file we are watching